maap/maap.py

```diff
@@ -195,6 +201,64 @@ def getJobStatus(self, jobid):
         )
         return response
 
+    def executeQuery(self, src, query={}, results=True, timeout=180, wait_interval=.5):
```

Comment:
What do you think about adding tests for this functionality?

Reply:
Done (added crude integration test)